std/math/trigonometry.d

```diff
@@ -237,14 +239,12 @@ SC17:   fprem1                          ;
 trigerr:
         jnp     Lret                    ; // if theta is NAN, return theta
         fstp    ST(0)                   ; // dump theta
-    }
-    return real.nan;
-
-Clear1: asm pure nothrow @nogc{
+        fld     rnan                    ; // return rnan
```

Comment:
How is this supposed to work? You're pushing rnan, then popping it immediately again via `fstp` in `Clear1` before returning, right?

Reply:
Ups, forgot to jump to `Lret` label. Nice catch, even though, it doesn't fail. Fixed